app/models/presenter/auction.rb

```diff
@@ -269,7 +88,7 @@ def markdown
                                             fenced_code_blocks: true,
                                             lax_spacing: true)
     end
-
+    
     def model
       __getobj__
     end
```

Comment:
My main metric was if it would change depending on the type of the auction, it should be moved into a `Policy::Auction` subtype (or the parent class). This meant moving the following types of methods:

1. Do I show the bids?
2. What bids do I display in various places?
3. How do I denote winning bids?
4. The auction status

These vary depending on if the auction is single-bid vs. multi-bid, or they call a method like `available?` which might vary for future auctions with a "Buy It Now" button for instance.

Since some of the bid display and other options are dependent on the current user (ie, you show the current user's bid if the user has made one), I decided to make the `Policy::Auction` class encapsulate a reference to a user. This meant I also moved over:

1. Methods that returned a user's bids
2. The `user_can_bid?` method
3. I added a new `show_bid_button?` method

So that's what happened. There are probably better ways of structuring this, but I would like the `Presenter::Auction` method to jsut contain helper methods for rendering auction fields and move the other logic to more appropriate objects.